modules/swagger-generator/src/main/java/io/swagger/v3/generator/model/LanguageOptions.java

```diff
@@ -79,4 +79,8 @@ public String clientsV3() {
     public String serversV3() {
         return serversV3.toString();
     }
-}
\ No newline at end of file
+
+    public boolean wereValuesSpecified(){
```

Comment:
see also comment below, as a standard file is always present, this will never return false, is that right?
I guess we need instead to either remove the availableOptions.yaml file and adapt the code (also for hiddenOptions I'd say, as we are now having some bad defaults there also), or equivalent mechanism